e2e/e2e_test.go

```diff
@@ -94,6 +96,22 @@ func Test_E2E_Only_Address_Memo(t *testing.T) {
 
 	// Step 2 - Verify the submitted topic messages
 	verifyTopicMessages(setupEnv, transactionResponse, 0, t)
+	//TODO: submit the tx send amount
+}
+
+func Test_E2E_Token_Transfer(t *testing.T) {
+	setupEnv := setup.Load()
+
+	memo := fmt.Sprintf("%s-0-0", receiverAddress)
+
+	// Step 1 - Verify the transfer of Hbars to the Bridge Account
+	transactionResponse, _ := verifyTokenTransferToBridgeAccount(setupEnv, memo, whbarReceiverAddress, t)
+
+	// Step 2 - Verify the submitted topic messages
+	verifyTopicMessages(setupEnv, transactionResponse, 0, t)
+	//TODO: submit the tx
```

Comment:
You have to add a new function here: `verifyTransferFromAPI` that would:
1. Query the validator (add new function in the `./clients/validator.go` for querying the transfer data)
2. Verify the returned data

After that you can add the next step: `SubmitAndVerifyMint`, which would use the returned data from the validator API and call the contract mint function. 
Wait for the transaction to be successfully mined.
Verify the token balance of the receiver